### DISCUSSION TITLE
#70 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by 0x-stan** [tog-ETH-er/togETHer#55] [General]
Agenda:

- Community Updates
- 下一任轮值主席提名
  > 轮值主席从正式成员中产生。前期，综合成员主观意愿、积极程度、组织能力等因素，『原语里弄』作为首任轮值主席。任何正式成员都可报名轮值主席。常规流程下，下一任轮值主席由当任轮值主席发起提名。[[source](https://docs.google.com/document/d/1RD491LNYSRvfe-CAU2wdWVqdT_Ru8gXJJj8bwKMAmpE/edit#)]
  - 欢迎各位社区来申请
 
会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：https://meet.google.com/zbv-cbra-hzu

**Comment by 0x-stan** [comment]
# Dapp-Learning

## sharing meeting

近两周举办了2场分享会
 
- 《DAO治理及案例分析》by Vivi
  - Youtube: https://youtu.be/axzax-UD65E
  - Bilibili: https://www.bilibili.com/video/BV1tP411p715

**Comment by Riely007** [comment]
### 以太坊翻译小组

- Translatathon 截止日期 31st, Aug

**Comment by luckyyang** [comment]
Rebase Community 
 
- web3 极客日报: 
  - 最近 14 期 http://daily.rebase.network/ 
- 招聘: 2 个团队发布 3 个招聘 https://github.com/rebase-network/who-is-hiring/issues
